extensions/database-documentation/package.json

```
@@ -0,0 +1,67 @@
+{
+	"name": "database-documentation",
+	"displayName": "%displayName%",
+	"description": "%description%",
+	"publisher": "Microsoft",
+	"version": "0.0.1",
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+	"engines": {
+		"vscode": "^1.25.0",
```

ADS-only extensions should just have an azdata engine version

```suggestion
		"vscode": "*",
```

---

This extension is also made to work on vscode.

---

Sounds good, although definitely make sure you're not using any newer APIs that only exist in higher engine versions as well. (both for VS Code and ADS)